sos/report/plugins/tpm2.py

```diff
@@ -0,0 +1,29 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, IndependentPlugin
+
```

Comment:
The flake8 linting test is failing because there should be 2 empty lines between the imports and the class definition here.

Reply:
Got it thanks updated now.